### PR TITLE
Sum employment and population

### PR DESCRIPTION
**Pull request by maggiesu0725**

### CODE REVIEW

**Comment by lhendey** [comment]
@maggiesu0725 - can you commit your changes?

**Comment by ptatian** [comment]
@lhendey I checked programs **Household income.sas**, **Tenure by Household income.sas**, **HHinc_owner_2000_2016.sas**, and **HHinc_renter_2000_2016.sas.** A few minor comments, but the code looks good!

**Comment by ptatian** [comment]
@lhendey There's a double backslash on line 83 of **Household income.sas**

File: Prog/2018/Washington region feature/cost burden for 80% median income.sas [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+data allstates;
+	set ipums.acs_2012_16_dc ipums.Acs_2012_16_md ipums.Acs_2012_16_va ipums.Acs_2012_16_wv ;
+	if puma in (101,102,103,104,105,1001,1002,1003,1004,1005,1006,1007,1101,1102,1103,1104,1105,
+				1106,1107,1301,1302,10701,10702,10703,51255,59301,59302,59303,59304,59305,59306,59307,59308,59309) then innercounty = 1;
+run;
+
+data allstates2;
+set allstates;
+	if (numprec = 1 and hhincome <= 108600*0.8*0.7) or (numprec=2 and hhincome <= 108600*0.8*0.8) or  (numprec=3 and hhincome <= 108600*0.8*0.9)
```
Comment (ptatian) [inline Prog/2018/Washington region feature/cost burden for 80% median income.sas]: @maggiesu0725 This isn't the way you should identify 80% AMI. We have a variable we create in all of the IPUMS files called _hud_inc_, that correctly categorizes households into the HUD income groups based on both income and size. 

hud_inc= | Label
-- | -- 
1 | Extremely low (0-30% AMI)
2 | Very low (31-50%)
3 | Low (51-80%) 
4 | Middle (81-120%)
5 | High (over 120%)

Sorry, I should have pointed that out. Can you just substitute what you have to use _hud_inc_ instead? Thank you. cc: @rpitingolo

**Comment by ptatian** [comment]
@maggiesu0725 @rpitingolo @lhendey Have all the changes for this pull request been committed? If so, we should merge and close the issue.

**Comment by lhendey** [comment]
@ptatian all of mine have. @rpitingolo @maggiesu0725

**Comment by maggiesu0725** [comment]
I don't have any other changes to commit. @ptatian @lhendey @rpitingolo

**Comment by ptatian** [comment]
@lhendey @maggiesu0725 @rpitingolo OK, I will go ahead and merge.

**Pull request merged**